js/main.js

```diff
@@ -36,4 +41,71 @@ function handleInputs(event) {
   data.entries.push(objectOfValues);
   $displayedImage.setAttribute('src', 'images/placeholder-image-square.jpg');
   $form.reset();
+
+  $dataViewEntries.classList.remove('hidden');
+  $dataViewEntryForm.classList.add('hidden');
+  data.view = 'entries';
+  location.reload();
```

Comment:
if the goal is to render the entries in real time, the best approach will be to call the function to render the entries, then use the prepend method to put the entry as the first child on the DOM in the ul.